release-notes/release-notes5013.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: TinyMCE 5.0.13
+title_nav: TinyMCE 5.0.13
+keywords: releasenotes newfeatures deleted technologypreview bugfixes knownissues
+---
+
+## Overview
+
+These Release Notes provide high-level coverage of: improvements and additions, known issues, important bug fixes & deprecated functionality associated with the release of TinyMCE 5.0.13.
+
+> **Note:** TinyMCE 5.0.10, 5.0.11, and 5.0.12 were not released to enterprise due to issues found during QA.
+> TinyMCE 5.0.13 enterprise contains all changes in the TinyMCE 5.0.10 through 5.0.13 community releases.
+
+## New features
+
+### TinyMCE
+
+#### Color map support
+
+TinyMCE now supports all valid HTML color formats for the `color_map` setting. For example, the following formats are now accepted:
+* `#FF33F9`
+* `rgb(255, 87, 51)`
+* `rgba(51, 79, 255, 0.8)`
+* `hsl(121, 80%, 60%)`
+* `blue`
+
+> **Note:** The previous hex code color format without the "#" is still accepted as valid values.
+
+For more information on the `color_map` setting, refer to the [documentation]({{site.baseurl}}/configure/content-appearance/#color_map).
+
+## Updates and enhancements
+
+### TinyMCE
+
+The updated TinyMCE 5.0.13 editor comes with numerous changes to the previous versions. A comprehensive list of the updates
+and fixed issues is available in the [changelog]({{site.baseurl}}/changelog/#version5013august62019).
+
+#### Accessibility improvements
+
+This release provides accessibility and keyboard navigation fixes for inline dialogs. Users will now be able to tab through the input and button elements in
+the dialog without losing cursor focus when closing the preferences menu.
```

Comment:
Actually, what do you mean by the "preferences menu"? That sounds like it's specific to the searchreplace dialog...

Reply:
Ahh good catch, in my notes I copied this from it was specifically about the search and replace dialog. We should definitely make this more generic.

Reply:
Maybe something like this?
```
This release provides accessibility and keyboard navigation fixes for inline dialogs. Users will now be able to tab through the input and button elements in the dialog without the cursor getting stuck or losing focus.
```

Reply:
updated by 400e59fdb2a7173667ec01a6572496e75139ebe5